### PR TITLE
create the database shcema

### PR DESCRIPTION
**Pull request by mossa-Sammer**
relates #3 
create the database schema in the build sql file

### CODE REVIEW

File: server/db/config/build.sql [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+
+CREATE TABLE services(
+  id SERIAL PRIMARY KEY NOT NULL,
+  name VARCHAR(255) NOT NULL
+);
+CREATE TABLE handyman(
+  handyman_id INTEGER PRIMARY KEY REFERENCES users(id),
+  job_title INTEGER REFERENCES services(id),
+  hour_rate INTEGER NOT NULL,
+  description VARCHAR(255) NOT NULL
```
Comment (fares98) [inline server/db/config/build.sql]: text not varchar

File: server/db/config/build.sql [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+);
+
+
+
+CREATE TABLE jobs (
+id SERIAL PRIMARY KEY NOT NULL, 
+client_id INTEGER REFERENCES users(id), 
+handyman_id INTEGER REFERENCES handyman(handyman_id), 
+description VARCHAR(255) NOT NULL,
```
Comment (fares98) [inline server/db/config/build.sql]: text not varchar

File: server/db/config/build.sql [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+
+
+
+CREATE TABLE jobs (
+id SERIAL PRIMARY KEY NOT NULL, 
+client_id INTEGER REFERENCES users(id), 
+handyman_id INTEGER REFERENCES handyman(handyman_id), 
+description VARCHAR(255) NOT NULL, 
+dead_line DATE NOT NULL, 
+price INTEGER NOT NULL,
```
Comment (fares98) [inline server/db/config/build.sql]: DEC instead of integer

File: server/db/config/build.sql [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+
+CREATE TABLE jobs (
+id SERIAL PRIMARY KEY NOT NULL, 
+client_id INTEGER REFERENCES users(id), 
+handyman_id INTEGER REFERENCES handyman(handyman_id), 
+description VARCHAR(255) NOT NULL, 
+dead_line DATE NOT NULL, 
+price INTEGER NOT NULL, 
+status VARCHAR(255) NOT NULL, 
+message VARCHAR(255),
```
Comment (fares98) [inline server/db/config/build.sql]: text

File: server/db/config/build.sql [hunk 5]
```diff
@@ -0,0 +1,42 @@
...
+CREATE TABLE services(
+  id SERIAL PRIMARY KEY NOT NULL,
+  name VARCHAR(255) NOT NULL
+);
+
+CREATE TABLE handyman(
+  handyman_id INTEGER PRIMARY KEY REFERENCES users(id),
+  job_title INTEGER REFERENCES services(id),
+  hour_rate INTEGER NOT NULL,
+  description VARCHAR NOT NULL
```
Comment (AbdallahAmmar96) [inline server/db/config/build.sql]:
```suggestion
  description text NOT NULL
```

File: server/db/config/build.sql [hunk 6]
```diff
@@ -0,0 +1,42 @@
...
+
+CREATE TABLE jobs (
+  id SERIAL PRIMARY KEY NOT NULL, 
+  client_id INTEGER REFERENCES users(id), 
+  handyman_id INTEGER REFERENCES handyman(handyman_id), 
+  description VARCHAR NOT NULL, 
+  dead_line DATE NOT NULL, 
+  price INTEGER NOT NULL, 
+  status VARCHAR(255) NOT NULL, 
+  message VARCHAR,
```
Comment (AbdallahAmmar96) [inline server/db/config/build.sql]:
```suggestion
  message text, 
```

File: server/db/config/build.sql [hunk 7]
```diff
@@ -0,0 +1,42 @@
...
+  id SERIAL PRIMARY KEY NOT NULL, 
+  client_id INTEGER REFERENCES users(id), 
+  handyman_id INTEGER REFERENCES handyman(handyman_id), 
+  description VARCHAR NOT NULL, 
+  dead_line DATE NOT NULL, 
+  price INTEGER NOT NULL, 
+  status VARCHAR(255) NOT NULL, 
+  message VARCHAR, 
+  street VARCHAR(255) NOT NULL, 
+  building_number VARCHAR(255) NOT NULL,
```
Comment (AbdallahAmmar96) [inline server/db/config/build.sql]:
```suggestion
  building_number INTEGER NOT NULL, 
```

File: server/db/config/build.sql [hunk 8]
```diff
@@ -0,0 +1,42 @@
...
+  job_title INTEGER REFERENCES services(id),
+  hour_rate INTEGER NOT NULL,
+  description TEXT NOT NULL
+);
+
+CREATE TABLE jobs (
+  id SERIAL PRIMARY KEY NOT NULL, 
+  client_id INTEGER REFERENCES users(id), 
+  handyman_id INTEGER REFERENCES handyman(handyman_id), 
+  description VARCHAR NOT NULL,
```
Comment (AbdallahAmmar96) [inline server/db/config/build.sql]:
```suggestion
  description TEXT NOT NULL, 
```

File: server/db/config/build.sql [hunk 9]
```diff
@@ -0,0 +1,42 @@
+BEGIN;
+
+DROP TABLE IF EXISTS users, handyman, services, jobs;
+
+CREATE TABLE users (
```
Comment (AbdallahAmmar96) [inline server/db/config/build.sql]:
```suggestion
CREATE TABLE users(
```

**Pull request merged**